test/controllers/ContinuedIllnessController.test.ts

```diff
@@ -0,0 +1,60 @@
+import { expect } from 'chai';
+import request from 'supertest';
+import { createApp } from '../ApplicationFactory';
+
+import { Appeal } from 'app/models/Appeal';
+import { ApplicationData } from 'app/models/ApplicationData';
+import { Illness } from 'app/models/Illness';
+import { Reasons } from 'app/models/Reasons';
+import { CONTINUED_ILLNESS_PAGE_URI } from 'app/utils/Paths';
+
+describe('ContinuedIllnessController', () => {
+
+    it('should show hint containing illness start date, and radio buttons for Yes and No on GET', async () => {
+        const  applicationData: Partial<ApplicationData> = {
+            appeal: {
+                penaltyIdentifier: {
+                    companyNumber: 'NI000000'
+                },
+                reasons: {
+                    illness: {
+                        illnessStart: new Date(2020, 4, 1)
+                    } as Illness
+                } as Reasons
+            } as Appeal,
+            navigation: { permissions: [CONTINUED_ILLNESS_PAGE_URI] }
+        };
+
+        const app = createApp(applicationData);
+
+        await request(app).get(CONTINUED_ILLNESS_PAGE_URI).expect(res => {
+            expect(res.text).to.include('class="govuk-hint"');
+            expect(res.text).to.include('You told us the illness started on 1 May 2020');
+            expect(res.text).to.include('type="radio"');
+            expect(res.text).to.include('value="yes"');
+            expect(res.text).to.include('value="no"');
+            const radioCount = (res.text.match(/type="radio"/g) || []).length;
+            expect(radioCount).to.equal(2);
+        });
+
+    });
+
+
+    it('should show an error if no option is selected on POST', async () => {
```

Comment:
Shall we add another one when validation passed to expect page reload?

Reply:
My feeling is that adding a test case for a temporary effect (rather than desired feature) wouldn't be a good idea but happy to do this if you feel it's necessary.

Reply:
Maybe that test should only check that 200 response is given? In a way that would test that page validation can passes.

Anyway just suggestion, I do not feel strong about that, can be added when navigation is implemented. Will live it up to your judgement.

Reply:
200 response check sounds like a good compromise, I'll implement that.